### PR TITLE
MOM of Community Meeting 26-01-2023

### PR DESCRIPTION
**Pull request by meenaljain0303**
Signed-off-by: meenaljain0303 <meenalj351@gmail.com>

# Pull Request Template

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

## Additional Context (Please include any Screenshots/gifs if relevant)

...

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings
- [ ] I have tagged the reviewers in a comment below incase my pull request is ready for a review
- [ ] I have signed the commit message to agree to Developer Certificate of Origin (DCO) (to certify that you wrote or otherwise have the right to submit your contribution to the project.) by adding "--signoff" to my git commit command.

### CODE REVIEW

File: working-groups/community_meeting/26-01-2023.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# 26-01-2023 [community]  
+## Agenda - Open to Discussion
+- GSoC & GSoD participation
+- Publications about moja global projects
+- Participation in conferences/meetups
+- Implementation of metrics to track progress
+
+## Discussion
+
+- Creating Blogs for the moja global organization. Like the technology on which we are working like Vuejs for frontend and all.
```
Comment (thisiskaransgit) [inline working-groups/community_meeting/26-01-2023.md:10]: Could you please change the technology discussed from Vuejs to "FLINT, FLINT.Cloud and more"

**Review by abhilipsasahoo03 — COMMENTED** [review]
Hi @meenaljain0303 can you add Gopi's suggestion to gamify contributions to Moja Global and automate the process of rewarding the long time contributors who have been making consistent contribution (leaderboard type stuff)? It was discussed as an alternative to DevRel position.

File: working-groups/community_meeting/26-01-2023.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+- Platform to write Blogs: Medium 
+
+##Reach out initiative :
+- College Outreach - Talks in college and collaborative work done by students (resources will be provided)
+- Conferences can be attended to write blog posts on Medium
+- Increase the reach of Moja Global
+- Communications team along with Harsh is processing the idea for creating online reach via Social Media.
+
+## Tips to keep in mind: 
+- Dont become an organiser, be the person to promote Moja Global
```
Comment (samyakshah3008) [inline working-groups/community_meeting/26-01-2023.md:22]: Great work @meenaljain0303 🚀 
I have a minor change to suggest here: 
- Can you please correct from `Dont` => `Don't`?

File: working-groups/community_meeting/26-01-2023.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+- GSoC & GSoD participation
+- Publications about moja global projects
+- Participation in conferences/meetups
+- Implementation of metrics to track progress
+
+## Discussion
+
+- Creating Blogs for the moja global organization. Like the technology on which we are working like Vuejs for frontend and all.
+- Example - Harsh is taking the blog on “DevOps practises” and “FLINT.Cloud GCP” by Gopi 
+- You being recognized by the name and able to learn lot’s of thing regarding the MojaGlobal just by writing about the same.
```
Comment (samyakshah3008) [inline working-groups/community_meeting/26-01-2023.md:12]: Can you please correct from `lot's of thing` => `lot's of things`? and on the same line from `MojaGlobal` to `moja global`?

File: working-groups/community_meeting/26-01-2023.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+- Platform to write Blogs: Medium
```
Comment (samyakshah3008) [inline working-groups/community_meeting/26-01-2023.md:13]: Can we provide Medium platform link as well?

File: working-groups/community_meeting/26-01-2023.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+- Implementation of metrics to track progress
+
+## Discussion
+
+- Creating Blogs for the moja global organization. Like the technology on which we are working like Vuejs for frontend and all.
+- Example - Harsh is taking the blog on “DevOps practises” and “FLINT.Cloud GCP” by Gopi 
+- You being recognized by the name and able to learn lot’s of thing regarding the MojaGlobal just by writing about the same.
+- Platform to write Blogs: Medium 
+
+##Reach out initiative :
```
Comment (samyakshah3008) [inline working-groups/community_meeting/26-01-2023.md:15]: Can you please remove the space after `initiative` so there is no gap between `:` and `Reach out initiative`

**Comment by BhaswatiRoy** [comment]
Hello @meenaljain0303 
Great work indeed. Ig there is a typo on line 27
image

File: working-groups/community_meeting/26-01-2023.md (hunk 4 above)
Comment (meenaljain0303) [reply working-groups/community_meeting/26-01-2023.md:13]: Publications will be published via either our personal accounts or a collective medium page of Moja Global. @HarshCasper can give more clarity regarding this.

**Pull request merged**